packages/backend/jobs-manager/service/src/modules/findings/commands/findings-handler-base.ts

```diff
@@ -38,8 +38,18 @@ export abstract class FindingHandlerBase<T extends FindingCommand>
     for (const sub of subs) {
       // Validate that, according to the conditions, the job should be executed.
       // If not, then we go straight for the other subscription
-      if (!SubscriptionsUtils.shouldExecute(sub.conditions, command)) continue;
-
+      if (
+        !SubscriptionsUtils.shouldExecute(
+          sub.isEnabled,
+          sub.conditions,
+          command,
+        )
+      ) {
+        this.logger.debug(
+          `Skipping job publication for ${sub.name}; conditions not met or subscription is disabled.`,
+        );
+        continue;
```

Comment:
Same here, we could get only the subscriptions that are enabled and skip some operations? Even though it is much lighter in the case of event subscriptions, it takes more resources than it not being returned by the service. The impact is lesser here than on cron subscriptions for sure. Food for thought.

Reply:
My thinking was the log usefulness outweighs the minimal performance impact. What would be more irritating than debugging why your job isn't starting and having no trace, haha